### ISSUE TITLE
Update background samples once UL become available

### ISSUE THREAD

**Issue by kmohrman** [TopEFT/topcoffea#99]
When we created the JSONs for the background samples, not all processes had a UL sample available. These processes are marked with a `NOTE: PLACEHOLDER` comment in the `make_json.py` file. Once the UL samples become available, we should replace these placeholders with the appropriate samples.

**Comment by kmohrman** [comment]
Only remaining non-v9 background samples:
* UL18_TTJets
* UL16APV_ZZTo4L

Currently using these dasgo queries to look for the missing samples:
```
dasgoclient -query "dataset=/TTJets_TuneCP5_13TeV-amcatnloFXFX-pythia8/*v9*UL*/NANOAODSIM"
```
```
dasgoclient -query "dataset=/ZZTo4L_TuneCP5_13TeV_powheg_pythia8/*v9**UL*/NANOAODSIM"
```

**Comment by kmohrman** [comment]
Closed by PR #217.
